pkg/kubectl/cmd/get.go

```diff
@@ -308,7 +306,9 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 	if err != nil {
 		return err
 	}
-
+	if r.TargetsSingleItems() {
```

Comment:
If I'm reading correctly, doesn't this remove the filter if I run `kubectl get svc/foo pods`?